rmgpy/rmg/react.py

```diff
@@ -54,13 +55,29 @@ def react(*spcTuples):
     Returns a flat generator object containing the generated Reaction objects.
     """
 
-    results = map_(
+    from rmgpy.rmg.main import maxProc
+    
+    # Available RAM memory (kB)
+    mem = psutil.virtual_memory()
+    user_mem = mem.available
+    
+    tmp = divmod(user_mem, resource.getrusage(resource.RUSAGE_SELF).ru_maxrss)
+    tmp2 = min(maxProc, tmp[0])
+    procNum = max(1, tmp2)
+    print 'For reaction generation {0} processes are used.'.format(procNum)
```

Comment:
We like to avoid print statements in favor of using the logging module. So you would just replace `print` with `logging.info()` or maybe `logging.debug` depending on whether you want this to be printed normally.

Something you should consider is how many times this will be printed (seems like it will be a lot) and whether it's useful info for a typical user.

Reply:
Done. It will be printed for each time we are spawning a pool of worker. It might be of interest as the number of processes/worker might differ from the user input, depending on the available RAM and the memory consumption of the RMG base process before spawning the workers.